src/Shouldly.Tests/ConventionTests/ShouldlyConventions.cs

```diff
@@ -1,3 +1,4 @@
+#if FullFramework
```

Comment:
I did some digging, it looks like the problem we are facing is this:
https://github.com/dotnet/corefx/issues/21079
and is fixed in 2.1:
https://github.com/dotnet/coreclr/pull/14696

It's also probably cleaner to keep the compiler constants to the functionality they toggle instead of being 1-1 with the tfm, cause it would be easy for someone to use this for something other than removing code that relies on symbols working for example.

Reply:
Added the stacktrace flag to .NET Standard 2.0 and all the tests passed

Reply:
> It's also probably cleaner to keep the compiler constants to the functionality they toggle instead of being 1-1 with the tfm, cause it would be easy for someone to use this for something other than removing code that relies on symbols working for example

Agreed. Now I've got the tests working I can refine these flags.

Reply:
Nice. What's weird for me is that I pull the branch into a fresh folder (proper fresh), do a restore, build, and test, and it cannot get the filenames (which are required by the tests to load the source), and I can see that the symbols are loaded.

I'm 100% hitting bug dotnet/corefx#21079, but clearly it's not an issue in AppVeyor
😕

Reply:
I was hitting the same, but as per the recommendation, if you copy the *.DiaSymReader.*.dll's from C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Roslyn to the tests bin directory, this temporarily solves the line number issues, and all the netcoreapp2.0 tests pass.

Reply:
@EddieGarmon oh nice, I'll have to give that a try. Thanks for the info 👍